apps/web/src/components/AddressMevLookup.tsx

```diff
@@ -11,7 +11,7 @@ import { socials, SocialType } from "@/constants/links";
 import { signInWithTwitter } from "@/services/supabase";
 import useSupabaseSession from "@/hooks/useSupabaseSession";
 import { LocalStorage } from "@/constants/storage";
-import clsx from "clsx";
+import { twMerge } from "tailwind-merge";
```

Comment:
lets use already existing `clsx` and `twMerge ` util `cn` from `utils/common`

```suggestion
import { twMerge } from "tailwind-merge";
```